msmtools/util/sorted_schur.py

```diff
@@ -259,6 +265,8 @@ def sorted_krylov_schur(P, m, z='LM', tol=1e-16):
         # Computes the error (based on the residual norm) associated with the i-th computed eigenpair.
         eigenval_error = E.computeError(i)
         top_eigenvals_error.append(eigenval_error)
+
+    # cut off excess dimensions also for the eigenvalues
```

Comment:
oh, I don't - I should have removed that comment, sorry! I left the extra eigenvalues to be able to check for splitting complex conjugate pairs in case we don't recompute the decomposition, but use a precomputed one.